datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/search/SearchUtils.java

```diff
@@ -75,7 +75,8 @@ private SearchUtils() {}
           EntityType.DATA_PRODUCT,
           EntityType.NOTEBOOK,
           EntityType.BUSINESS_ATTRIBUTE,
-          EntityType.SCHEMA_FIELD);
+          EntityType.SCHEMA_FIELD,
+          EntityType.DATA_PROCESS_INSTANCE);
```

Comment:
This will make data process instances show up in search / scroll across entities calls when no entity types are provided. Let's instead keep this out of this location so that data process instances are not returned in primary search.